operator/v1/types_console.go

```diff
@@ -174,6 +178,15 @@ type DeveloperConsoleCatalogCategoryMeta struct {
 	Tags []string `json:"tags,omitempty"`
 }
 
+type AccessReviewResources struct {
+	// kind is a string value representing the REST resource.
```

Comment:
Maybe give a few examples of expected kinds?

Reply:
Removed this type and using the existing `ResourceAttributes` type